README.md

```diff
@@ -31,7 +25,7 @@ You can install the client in your project by using composer:
 composer require elastic/app-search
 ```
 
-## Versioning
+### Versioning
```

Comment:
Are you sure you want to increment the title level.
All other sections referenced in the TOC are h2, so it would be more consistent to keep the previous level IMHO.